tests/integration/fuzzing/fuzzing_test.py

```diff
@@ -122,3 +122,24 @@ async def test_should_not_share_state(
         testing_summary,
         expected_passed_test_cases_names=["test_context"],
     )
+
+
+async def test_parameterized_with_examples_tests(
+    run_cairo_test_runner: RunCairoTestRunnerFixture,
+):
+    testing_summary = await run_cairo_test_runner(
+        Path(__file__).parent / "parameterized_test.cairo"
+    )
+
+    assert_cairo_test_cases(
+        testing_summary,
+        expected_passed_test_cases_names=[
+            "test_examples",
+            "test_examples_without_given",
+        ],
+    )
+
+    assert len(testing_summary.passed) == 2
+    passed_set = {passed.fuzz_runs_count for passed in testing_summary.passed}
+    assert 2 in passed_set
+    assert 7 in passed_set
```

Comment:
this will be unneeded when you assert exact values in test case Cairo code

Reply:
I don't know, I'd leave this but if you see a redundancy here, I'm ok with removing any of these checks.